dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/accounting/Financials.scala

```diff
@@ -0,0 +1,17 @@
+package org.bitcoins.dlc.wallet.accounting
+
+import org.bitcoins.core.protocol.transaction.Transaction
+import org.bitcoins.dlc.wallet.models.{DLCAcceptDb, DLCDb, DLCOfferDb}
+
+case class Financials(
```

Comment:
nit: imo this name doesn't really reflect what it contains.

Maybe `DLCAccountingDbs`